### ISSUE TITLE
Perform hyperparameter tuning

### ISSUE THREAD

**Issue by SRshohan** [SRshohan/PiFaceChain#13]
learning rate, batch size, etc.
